articles/container-registry/container-registry-transfer-images.md

```diff
@@ -421,7 +421,8 @@ az resource delete \
   * Not all artifacts, or none, are transferred. Confirm spelling of artifacts in export run, and name of blob in export and import runs. Confirm you are transferring a maximum of 50 artifacts.
   * Pipeline run might not have completed. An export or import run can take some time. 
   * For other pipeline issues, provide the deployment [correlation ID](../azure-resource-manager/templates/deployment-history.md) of the export run or import run to the Azure Container Registry team.
-
+* **Problems pulling the image in an airgapped environment**
+  * If you see errors regarding foreign layers or attempts to resolve mcr.microsoft.com when attempting to pull an image in an airgapped environment, your image manifest likely has non-distributable layers. Due to the nature of an airgapped environment, these images will often fail to pull. You can confirm that this is the case by checking the image manifest for any references to external registries. If this is the case, you will need to push the non-distributable layers to your public cloud ACR prior to deploying an export pipeline-run for that image. For guidance on how to do this, see [How do I push non-distributable layers to a registry?](./container-registry-faq.md#how-do-i-push-non-distributable-layers-to-a-registry)
```

Comment:
The content/format looks fine. One suggestion is to substitute "physically isolated" or similar for "airgapped", as a more generic term. Thx.

Reply:
done, sorry about the delay